setup.py

```diff
@@ -53,8 +53,6 @@
         "Topic :: Software Development",
     ],
     entry_points={
-        "console_scripts": [
-            "trlc = trlc.trlc:main",
-        ],
+        "console_scripts": ["trlc = trlc.trlc:main"],
```

Comment:
Was this a style change by some formatter tool? It is not really necessary, is it?

Reply:
no, it was intentional to be consistent with the format of the other setup.py files and because no line break is needed here